src/storage/src/hummock/compactor/shared_buffer_compact.rs

```diff
@@ -260,7 +260,7 @@ pub async fn merge_imms_in_memory(
 
     let mut imm_iters = Vec::with_capacity(imms.len());
     for imm in imms {
-        assert!(imm.kv_count() > 0, "imm should not be empty");
+        assert!(imm.kv_count() > 0 || imm.has_range_tombstone(), "imm should not be empty");
```

Comment:
any test case ?

Reply:
I have a test case that can reproduce, but it seems not suitable to add it to e2e tests.